hana/get_platform.sls

```diff
@@ -0,0 +1,18 @@
+#!py
+import platform
+
+PLATFORMS = {
+    'x86_64': 'HDB_CLIENT_LINUX_X86_64',
+    'ppc64': 'HDB_CLIENT_LINUX_PPC64',
```

Comment:
Thanks for your comment! 
I saw that SAP provides the installer for ppc64, so that's why I added it here.
We can remove it @diegoakechi what do you think?

Reply:
@arbulu89 Yes, it is not supported by SUSE and AFAIK it is not supported by SAP anymore. It is there for backward compatibility. We can remove it.